config/settings.yml

```diff
@@ -109,6 +109,7 @@
   :history:
     :keep_events: 6.months
     :purge_window_size: 1000
+  :publish_events: true
```

Comment:
wondering if the term `syndicate` would be better?  `syndicate_events`?

(naming is hard)

Reply:
:+1: done